src/components/NavBar.jsx

```diff
@@ -1,16 +1,42 @@
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 import './NavBar.css';
+import PropTypes from 'prop-types';
+import Head from '../img/head.png';
 
-const NavBar = () => (
-  <header className="header">
-    <>
-      <h1>Book store</h1>
-      <ul>
-        <li><Link to="/">Home</Link></li>
-        <li><Link to="/categories">Categories</Link></li>
-      </ul>
-    </>
-  </header>
-);
+const NavBar = (props) => {
+  const { active } = props;
+  const { home, cat } = active;
+  return (
+    <header className="header">
+      <>
+        <div className="logo-menu">
```

Comment:
- [optional] great job. Please kindly align the main title and the user icon placeholder with the page like in the original design. Please try as much as possible to keep to the details.
yours
![Screenshot_3](https://github.com/wacoo/react-book-store/assets/65298048/0327042d-5968-45ca-bd17-35ced7dd2168)
original
![Screenshot_4](https://github.com/wacoo/react-book-store/assets/65298048/4e7230fd-ce8a-4387-bf9f-f5881703d03e)